_posts/events/2019-08-02-mwe.md

```diff
@@ -0,0 +1,95 @@
+ * Processing of MWEs and constructions in syntactic and semantic frameworks (e.g. CCG, CxG, HPSG, LFG, TAG, UD, etc.), and in end-user applications (e.g. information extraction, machine translation and summarization)
+ * Original discovery and identification methods for MWEs and constructions
+ * MWEs and constructions in language acquisition and in non-standard language (e.g. tweets, forums, spontaneous speech)
+ * Evaluation of annotation and processing techniques for MWEs and constructions 
+ * Retrospective comparative analyses from the PARSEME shared tasks on automatic identification of MWEs
+
+Note that, with the intention to also perpetuate previous converging effects with the Construction Grammar community (see the LAW-MWE-CxG 2018 workshop), we extend the traditional MWE scope to include grammatical constructions.
+
+SUBMISSION MODALITIES
+
+There are two tracks:
+ * Regular research track, where the submissions must be substantially original.
+ * Dissemination track, which welcomes recent previously published work (or work accepted for publication), dedicated explicitly both to MWEs and WordNet.   This will be presented to encourage discussion, but only the abstract will appear in the proceedings.
+
+The regular research track submissions should follow one of the 2 formats:
+ * Long papers (8 content pages + references): Long papers should report on solid and finished research including new experimental results, resources and/or techniques.
+ * Short papers (4 content pages + references): Short papers should report on small experiments, focused contributions, ongoing research, negative results and/or philosophical discussion. 
+
+The decisions as to oral or poster presentations of the selected papers will be taken by the PC chairs. No distinction between papers presented orally and as posters is made in the workshop proceedings. There is no limit on the number of reference pages. Authors will be granted an extra page for the final version of their papers. The submission will be double-blind, as understood by the ACL 2019 submission policy. The reported research should be substantially original. Papers available as preprints can also be submitted provided that they fulfil the conditions defined by the new ACL Policies for Submission, Review and Citation. For both types of submissions in this track, the ACL 2019 templates should be used. 
+
+The dissemination track submissions are not anonymous, and they should not exceed one page, including the authors' names and affiliations, the mention of the original venue, the link to the original paper and a short explanation why the paper is relevant to MWEs and Wordnets workshop. If the original paper is not publicly available, it should also be submitted in a separate .pdf file but it does not have to follow the ACL 2019 template.
+
+All papers should be submitted via the following START space: https://www.softconf.com/acl2019/mwewn
```

Comment:
Put `<` and `>` around the URL so that it is rendered as a link.

Reply:
Done